app/assets/javascripts/dashboard/dataset/import_general_component.js

```diff
@@ -121,6 +121,14 @@ export default class ImportGeneralComponent extends React.PureComponent<Props, S
             </FormItemWithInfo>
           </Col>
         </Row>
+        <FormItemWithInfo
+          label="Publication Date"
+          info="This date is the publication date of the dataset. It is used to sort the datasets in the dataset gallery."
```

Comment:
I'd avoid "publication date" here as this might make users think that the dataset is public.

How about:

label: "Sorting Date"
info: "This date can be used to sort the datasets within webKnossos. For example, if the dataset was published in a paper, you can input the publication date of the paper here."

@normanrz @daniel-wer Do you agree?

Reply:
We already had a discussion about this naming issue. But I think your solution is more elegant and forward.